packages/react-aria-components/src/Tabs.tsx

```diff
@@ -37,7 +37,7 @@ export interface TabListProps<T> extends StyleRenderProps<TabListRenderProps>, A
 export interface TabListRenderProps {
   /**
    * The orientation of the tab list.
-   * @selector [aria-orientation="horizontal | vertical"]
+   * @selector [aria-orientation="horizontal | vertical"], [data-orientation="horizontal | vertical"]
```

Comment:
im aware that this doesn't wrap in the `PropsTable` in the docs so if anyone has any suggestions that would be much appreciated. in the meantime, i'll continue messing around with it

Reply:
I'd say we should just drop the aria versions from the docs, and only document the data attributes.

Reply:
https://github.com/adobe/react-spectrum/blob/0e359399ade3d2047b6e6ef4de5ec6e041f8689b/packages/dev/docs/src/docs.css#L434-L436C2

looks like it's related to this line